pydantic_ai_slim/pydantic_ai/agent/__init__.py

```diff
@@ -292,6 +303,7 @@ def __init__(
 
         self._output_type = output_type
         self.instrument = instrument
+        self._metadata = metadata
```

Comment:
Not sure about the sunder, and that it's not represented in repr. I went by feeling.